modules/nf-core/crisprcleanr/normalize/main.nf

```diff
@@ -0,0 +1,52 @@
+process CRISPRCLEANR_NORMALIZE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::r-crisprcleanr=3.0.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/r-crisprcleanr:3.0.0--r42hdfd78af_1':
+        'quay.io/biocontainers/r-crisprcleanr:3.0.0--r42hdfd78af_1' }"
+
+    input:
+    tuple val(meta), path(count_file), path(library_file)
+    val(min_reads)
+    val(min_targeted_genes)
+
+    output:
+    tuple val(meta), path("*_norm_table.tsv"), emit: norm_count_file
```

Comment:
Could you align the emits? There are folks around here who care about such things :-D